beets/library.py

```diff
@@ -658,6 +659,11 @@ def relation_join(cls) -> str:
             f"ON {cls._table}.album_id = {cls._relation._table}.id"
         )
 
+    @property
+    def filepath(self) -> Path | None:
+        """The path to the item's file as pathlib.Path."""
+        return Path(os.fsdecode(self.path)) if self.path else self.path
+
```

Comment:
Yesss pathlib!!!

Reply:
Ultimately we will want to adjust `PathType` here I think (right, @Serene-Arc?)

```python
    def from_sql(self, sql_value) -> Path | None:
        value = self.normalize(sql_value)
        return Path(os.fsdecode(value)) if value else value
```

but it's too early yet since the entire codebase is expecting this to return `bytes` 😅 .

Thus this `filepath` property kind of allows us to migrate to `filepath.Path` step-by-step 😃